json-endpoints/jetpack/class.jetpack-json-api-plugins-endpoint.php

```diff
@@ -183,4 +189,44 @@ protected function get_plugin_updates( $plugin_file ) {
 		}
 		return null;
 	}
+
+	/**
+	 * Get custom action link tags that the plugin is using
+	 * Ref: https://codex.wordpress.org/Plugin_API/Filter_Reference/plugin_action_links_(plugin_file_name)
+	 * @return array of plugin action links (key: link name value: url)
+	 */
+	 protected function get_plugin_action_links( $plugin_file ) {
+		 $formatted_action_links = array();
+
+		 $action_links = array();
+		 /** This filter is documented in src/wp-admin/includes/class-wp-plugins-list-table.php */
+		 $action_links = apply_filters( 'plugin_action_links', $action_links, $plugin_file, null, 'all' );
+		 /** This filter is documented in src/wp-admin/includes/class-wp-plugins-list-table.php */
+		 $action_links = apply_filters( "plugin_action_links_{$plugin_file}", $action_links, $plugin_file, null, 'all' );
+		 if ( count( $action_links ) > 0 ) {
+			 $dom_doc = new DOMDocument;
```

Comment:
No, I don't think we have any means of knowing that percentage. My hope is that it will only affect sites that are running on manually configured servers. Thanks for adding that change!